app/routers/weight.py

```diff
@@ -0,0 +1,65 @@
+from typing import Union
+
+from fastapi import APIRouter, Depends, Request
+from sqlalchemy.orm import Session
+from starlette.responses import RedirectResponse
+
+from app.database.models import User
+from app.dependencies import get_db
+from app.dependencies import templates
+
+
+router = APIRouter(tags=["weight"],)
+
+
+@router.get("/weight")
+async def weight_form(
+        request: Request,
+        session: Session = Depends(get_db),
+        target: Union[float, None] = None,
+        current_weight: Union[float, None] = None,
+        ):
+    user = session.query(User).filter_by(id=1).first()
```

Comment:
Use real user_id

Reply:
I add a TODO for waiting for user registration